lutris/util/jobs.py

```diff
@@ -15,12 +15,12 @@ def __init__(self, func, callback, *args, **kwargs):
         """
         self.callback_task = None
         self.stop_request = threading.Event()
-
-        super().__init__(target=self.target, args=args, kwargs=kwargs)
         self.function = func
         self.callback = callback if callback else lambda r, e: None
-        self.daemon = kwargs.pop("daemon", True)
 
+        super().__init__(target=self.target, args=args, kwargs=kwargs)
+        #self.daemon = kwargs.pop("daemon", True)
+        self.setDaemon(kwargs.pop("daemon", True))
```

Comment:
I think this change is wrong; we support back to Python 3.7, and by then setDaemon had been deprecated. You are supposed to set the self.daemon property.

So why setDaemon?

Also, moving the call to ``super().__init__`` later seems harmless, but why?

Reply:
I'll change it back sorry, I was not aware. 

For moving the call to `super().__init__` later , honestly I just liked how it looked better. I can revert it back too if you want.